src/fhirdefs/load.ts

```diff
@@ -113,6 +113,51 @@ export async function loadDependency(
   return FHIRDefs;
 }
 
+/**
+ * Loads custom resources defined in ig-data into FHIRDefs
+ * @param {string} input - The input path to the cli
+ * @param {FHIRDefinitions} defs - The FHIRDefinitions object to load definitions into
+ */
+export function loadCustomResources(input: string, defs: FHIRDefinitions): void {
+  // Similar code for loading custom resources exists in IGExporter.ts addPredefinedResources()
+  const pathEnds = [
+    'capabilities',
+    'extensions',
+    'models',
+    'operations',
+    'profiles',
+    'resources',
+    'vocabulary',
+    'examples' // Must come last in case examples are of other resources
```

Comment:
I thought we decided _not_ to load examples up front.  I'm guessing you need to do this since we also decided to put the file format checks up front?

Reply:
Also -- I don't think that comment is relevant anymore.  It's fine for `examples` to be last, but we no longer need to document a specific reason for it.

Reply:
Yeah I left it in for the checks of xml and such.